io.github.supercollider.SuperCollider.yml

```diff
@@ -0,0 +1,66 @@
+app-id: io.github.supercollider.SuperCollider
+runtime: org.kde.Platform
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15-22.08'
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: scide
+rename-icon: supercollider
+rename-icon: sc_ide
+rename-desktop-file: SuperColliderIDE.desktop
+
+modules:
+  - name: SuperCollider
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DSC_EL=no
+    sources:
+      - type: archive
+        url: https://github.com/supercollider/supercollider/releases/download/Version-3.13.0/SuperCollider-3.13.0-Source.tar.bz2
+        sha256: 0fc5dba5baeae37f9069ad28885aa405c6a25309e3886cbe111bd3b7c05531ce
+  - name: fftw3
+    buildsystem: autotools
+    config-opts:
+      - "--enable-shared"
+      - "--enable-float"
+    sources:
+      - type: archive
+        url: http://fftw.org/fftw-3.3.10.tar.gz
+        sha256: 56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467
+  - name: sc3-plugins
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DFFTW3F_LIBRARY=/app/lib/libfftw3f.so"
+    sources:
+      - type: archive
+        url: https://github.com/supercollider/sc3-plugins/releases/download/Version-3.13.0/sc3-plugins-3.13.0-Source.tar.bz2
+        sha256: f8deeb861d402e2a72db51d40b48c443690261b5a53af78983d4cf7ba76717a2
+  - name: git
+    make-args:
+      - NO_TCLTK=YesPlease
+      - NO_PERL=YesPlease
+      - INSTALL_SYMLINKS=1
+    make-install-args:
+      - NO_TCLTK=YesPlease
+      - NO_PERL=YesPlease
+      - INSTALL_SYMLINKS=1
+    post-install:
+      - find . -type f -name perllocal.pod -delete
+    sources:
+      - type: archive
+        url: https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.39.0.tar.xz
+        sha256: ba199b13fb5a99ca3dec917b0bd736bc0eb5a9df87737d435eddfdf10d69265b
+finish-args:
+  # We need MIDI, Audio, USB, DRI, and realtime performances
+  - --device=all
```

Comment:
midi and sound aren't handle by this. As for realtime the use of portals is the only way.